collagraph/__init__.py

```diff
@@ -170,6 +181,31 @@ def perform_unit_of_work(self, fiber: Fiber) -> Optional[Fiber]:
                 return sibling
             next_fiber = next_fiber.parent
 
+    def update_class_component(self, fiber: Fiber):
+        component = fiber.component or fiber.alternate and fiber.alternate.component
+        if not component:
+            component = fiber.type(fiber.props)
+
+        # Attach the component instance to the fiber
+        fiber.component = component
+        fiber.component_watcher = watch(
+            # FIXME: sometimes fiber.component is None
+            # Seems to happen after a while after generating lots of changes.
+            # Could be that it is only when updates are coming in faster than
+            # they are rendered...
```

Comment:
is this still an issue? can you reproduce it?

Reply:
Probably not with the current example, but I'm sure it still exists. I'll make an issue about this such that we don't forget about it.

Reply:
I've created #17 to keep track of this.